src/main/java/gr/cytech/chatreminderbot/rest/controlCases/CaseSetConfigurations.java

```diff
@@ -0,0 +1,52 @@
+package gr.cytech.chatreminderbot.rest.controlCases;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CaseSetConfigurations {
+    @PersistenceContext(name = "wa")
+    public EntityManager entityManager;
+
+    private ArrayList<String> splitMsg;
+
+    public void setSplitMsg(ArrayList<String> splitMsg) {
+        this.splitMsg = splitMsg;
+    }
+
+    @Transactional
+    public String configurationController() {
+        if (splitMsg.size() == 1) {
+            return configCommand();
+        }
+
+        if (splitMsg.get(1).equals("buttonUrl") && splitMsg.size() == 3) {
```

Comment:
why must you check that it's `buttonUrl` and treat it specially?
isn't it viable to handle this in a generic/abstract way?